.npmrc

```diff
@@ -1 +1,3 @@
 publish-branch=iroha2
+# FILE: .npmrc
+@iroha2:registry=https://nexus.iroha.tech/repository/npm-group/
```

Comment:
This line is for those who is _using_ SDK's published packages. Id doesn't make sense to be here, so, remove.